src/commands/branch.js

```diff
@@ -19,7 +20,8 @@ class BranchCommand extends AbstractAuthenticatedCommand {
     } catch (error) {
       const customError = BranchManager.handleBranchError(error);
 
-      return this.error(customError);
+      logger.error(customError);
+      return this.exit(2);
```

Comment:
[Misc] Why did you make an `exit(2)` here?

Reply:
Same question for all `exit(2)` across this file

Reply:
As discussed, it is the default exit code thrown by `this.error`...
So to stay iso and satisfy the current tests, I had to specify the exit 2.